main.cpp

```diff
@@ -0,0 +1,133 @@
+#include <iostream>
+#include <fstream>
+
+using std::cin;
+using std::cout;
+using std::endl;
+using std::string;
+
+void init(){
+    string first_name = "";
+    do {
+        cout << "Enter your first name:" << endl;
+        getline (cin, first_name);
+    } while (first_name == "");
+
+    string last_name = "";
+    do {
+        cout << "Enter your last name:" << endl;
+        getline (cin, last_name);
+    } while (last_name == "");
+
+    string email_address = "";
+    do {
+        cout << "Enter your email address:" << endl;
+        getline (cin, email_address);
+    } while (email_address == "");
+
+    string cypher = "";
+    do {
+        cout << "Enter a unique cypher:" << endl;
+        getline (cin, cypher);
+    } while (cypher == "");
+
+    string time_offset = "";
+    do {
+        cout << "Enter your Time Zone offset:" << endl;
+        getline (cin, time_offset);
+    } while (time_offset == "");
+
+    string file_path = "";
+    cout << "Enter path to knownrecipients file:" << endl;
+    getline (cin, file_path);
+    if (file_path == "") {
+        file_path = "knownrecipients.txt";
+    }
+
+    std::ofstream output_file_init;
+    output_file_init.open(file_path);
+
+    output_file_init << first_name << endl;
+    output_file_init << last_name << endl;
+    output_file_init << email_address << endl;
+    output_file_init << cypher << endl;
+    output_file_init << time_offset << endl;
+
+    output_file_init.close();
+}
+
+void edit(){
```

Comment:
You have to define them ahead of usage because C++ is very procedural. After the test we will talk about function prototypes and header files so we can put the functions wherever we want them. Including the header files :)